mmv1/products/bigquery/Job.yaml

```diff
@@ -859,6 +871,11 @@ properties:
                 type: String
                 description: 'The ID of the model.'
                 required: true
+      - name: 'reservation'
```

Comment:
Add `min_version: beta` for this property too if it's meant for a public preview release to the `google-beta` provider.